biz.ganttproject.core/src/biz/ganttproject/core/chart/scene/ResourcesColumnSceneBuilder.kt

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020 Dmitry Kazakov, BarD Software s.r.o
+
+This file is part of GanttProject, an open-source project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package biz.ganttproject.core.chart.scene
+
+import biz.ganttproject.core.chart.canvas.Canvas
+import biz.ganttproject.core.chart.canvas.TextMetrics
+import java.awt.Color
+
+private const val OFFSET = 5
```

Comment:
This doesn't look as an offset of any sort, no? Just a default width of the empty list, divided by 2. I mean it is used only for this purpose.

Reply:
It is meant to be value of left and right border offset

Reply:
Oh, I forgot to use this instead of the number when creating text